<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#main_widget
{

background-color: rgb(245, 245, 244);
}



</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="header_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-7</y>
         <width>581</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>28</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 28pt &quot;Arial&quot;;
color: rgb(255, 255, 255);
background-color: rgb(128, 60, 224);</string>
       </property>
       <property name="text">
        <string>TrafficRecord</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="sign_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Arial&quot;;
color:#803ce0;</string>
       </property>
       <property name="text">
        <string>Usernmae</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Arial&quot;;
color:#803ce0;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="user_name">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>381</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:#F8EAFF;
border:2px solid #803CE0;
border-radius:10px;
color:#803CE0;
font-family:Arial;
font:10pt;
}

QLineEdit:focus
{
border:2px solid #FBAD25;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="user_password">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>381</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:#F8EAFF;
border:2px solid #803CE0;
border-radius:10px;
color:#803CE0;
font-family:Arial;
font:10pt;
}

QLineEdit:focus
{
border:2px solid #FBAD25;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="push_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-color: rgb(128, 60, 224);
border-radius:10px;
font:11pt Arial;
color:white;
}


QPushButton:hover
{
background-color:rgb(193, 81, 241);
}</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
